q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -40,52 +41,38 @@ def sniff(self):
 
 
 class TaxonomicClassifierDirFmt(model.DirectoryFormat):
-    preprocess_params = model.File('preprocess_params.json', format=JSONFormat)
     sklearn_pipeline = model.File('sklearn_pipeline.tar', format=PickleFormat)
 
 
 # Transformers
 @plugin.register_transformer
-def _1(dirfmt: TaxonomicClassifierDirFmt) -> dict:
+def _1(dirfmt: TaxonomicClassifierDirFmt) -> Pipeline:
```

Comment:
Make sure to update the transformers to use the new dir fmt discussed above. Lastly, add a new trivial transformer for handling `TaxonomicClassifierDirFmt`.